### PR TITLE
Doc

### PR DESCRIPTION
**Pull request by AndreBiedenkapp**
This pull request does not contain any html files, just the .rst files and the neccessary files for building the github-pages

### CODE REVIEW

File: doc/conf.py [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+source_suffix = '.rst'
+
+# The encoding of source files.
+#source_encoding = 'utf-8-sig'
+
+# The master toctree document.
+master_doc = 'index'
+
+# General information about the project.
+project = u'SMAC3'
```
Comment (mfeurer) [inline doc/conf.py]: In the setup.py the project is called only smac. I'm not sure which one is appropriate.

File: doc/conf.py [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+copyright = u'2015, Marius Lindauer, Stefan Falkner, Katharina Eggensperger, Aaron Klein, Frank Hutter'
```
Comment (mfeurer) [inline doc/conf.py]: Maybe these should be moved to the __init__ file in the smac package, then we can also import this information in setup.py and it is kept at a single place.

File: doc/conf.py [hunk 3]
```diff
@@ -0,0 +1,290 @@
...
+author = u'Marius Lindauer, Stefan Falkner, Katharina Eggensperger, Aaron Klein, Frank Hutter'
+
+# The version info for the project you're documenting, acts as replacement for
+# |version| and |release|, also used in various other places throughout the
+# built documents.
+#
+# The short X.Y version.
+version = '0.0.1dev'
```
Comment (mfeurer) [inline doc/conf.py]: Same here, the setup.py currently lists version 0.1.3

File: examples/leadingones.py [hunk 4]
```diff
@@ -70,7 +61,7 @@ def leading_ones(cfg, seed):
 
 stats = Stats(scenario)
 
-# register function to be optimize
+# register function to be optimized
 taf = ExecuteTAFunc(leading_ones, stats)
```
Comment (mfeurer) [inline examples/leadingones.py]: To comply to the latest changes you must add `run_objective="quality"`.

File: doc/conf.py (hunk 1 above)
Comment (AndreBiedenkapp) [inline doc/conf.py]: In the newest commit I set this to SMAC to have it as in the setup.py

File: doc/conf.py (hunk 3 above)
Comment (AndreBiedenkapp) [inline doc/conf.py]: Fixed in the newest commit

File: examples/leadingones.py (hunk 4 above)
Comment (AndreBiedenkapp) [inline examples/leadingones.py]: Fixed in the newest commit

**Comment by AndreBiedenkapp** [comment]
The duplicate rst files have been removed from the source folder

**Comment by mfeurer** [comment]
Thanks for working on the requested changes. The documentation doesn't build like this because config file thinks that the rst files are still in the source directory. Also, can you please fix the warnings when building the documentation?

**Comment by AndreBiedenkapp** [comment]
I fixed the sphinx warnings but I'm not sure what you mean with the config file things the rst files are in the source directory.
I've always build the documentation with the following command 'sphinx-build doc/ doc/build'

**Comment by mfeurer** [comment]
> I fixed the sphinx warnings

Thanks

> I'm not sure what you mean with the config file things the rst files are in the source directory.

Me neither any more, the PR looks good (from a technical perspective. I didn't go through the content).

Now there is only the merge conflict. Could you please fix this so we can merge your PR?

**Comment by mlindauer** [comment]
Hi,

@AndreBiedenkapp could you please also check that the docu is still consistent with the dev branch. We added last week new facades for SMAC and one to optimize a Python function. The examples are updated accordingly.

Best,
Marius

**Comment by AndreBiedenkapp** [comment]
@mfeurer I fixed the merge conflicts.
@mlindauer I updated the documentation where necessary.

**Pull request merged**